Whats new for August

BillWagner opened the pull request:
Create the what's new page for August.

BillWagner committed:
run automated tool

In addition to running the tool:

- remove February
- update TOC
- update index.

BillWagner committed:
Remove vendor from contributor list

BillWagner commented:
closing and opening for a fresh build

Youssef1313 commented on docs/whats-new/dotnet-2020-08.md:
```diff
@@ -0,0 +1,143 @@
+---
```
I believe you started to remove the "dotnet" prefix from file names.

Youssef1313 commented on docs/whats-new/dotnet-2020-08.md:
```diff
@@ -0,0 +1,143 @@
+### Updated articles
+
+- [Compiler Warning (level 1) CS4014](/dotnet/csharp/language-reference/compiler-messages/cs4014) - Update async concept articles in the C# programming guide
+- [Sorry, we don't have specifics on this C# error](/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error) - F1 keywords
```
This shouldn't probably be listed.

BillWagner committed:
remove generic warning

This shouldn't be listed. What changed was a list of metadata for warnings that lack pages.

mairaw commented on docs/whats-new/2020-08.md:
```diff
@@ -0,0 +1,142 @@
+- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
+- [JohyPark](https://github.com/JohyPark) - Joohyung Park (6)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (6)
+- [sughosneo](https://github.com/sughosneo) - Sumit Ghosh (6)
```
new vendor on the team

mairaw commented on docs/whats-new/2020-08.md:
```diff
@@ -0,0 +1,142 @@
+- [akashmanoj213](https://github.com/akashmanoj213) - Akash Manoj (1)
+- [anadem](https://github.com/anadem) - Alan Sinclair (1)
+- [angularsen](https://github.com/angularsen) - Andreas Gullberg Larsen (1)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (1)
```
this in theory is probably an employee behind a bot?

BillWagner replied:
thanks. I have made a code change to fix this, but we haven't deployed it yet.

IEvangelist reviewed (approved):
Looks good to me, :shipit: when you're ready.

BillWagner committed:
remove bot and vendor
